jenkins/opensearch/distribution-build.jenkinsfile

```diff
@@ -12,6 +12,11 @@ pipeline {
             description: 'Input manifest under the manifests folder, e.g. 2.0.0/opensearch-2.0.0.yml.',
             trim: true
         )
+        string(
+            name: 'DISTRIBUTION',
+            description: 'Distribution to build, e.g. tar, zip, and rpm.',
+            trim: true
+        )
```

Comment:
I wonder do we want this parameter so each distribution build would be specific to one distribution? or we run build for both tarball and rpm in parallel within one job.

Reply:
I am more for doing within one job. See https://github.com/opensearch-project/opensearch-build/pull/1807#issuecomment-1075851261

Reply:
Oh I missed that part. Thanks for explanation :D